.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file can be removed.

You can delete it, push the changes to this branch and re-open Eclipse. The file should be regenerated. Or you can move to somewhere else and bring it back after merging.